<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
  <class>QmitknnUNetToolGUIControls</class>
  <widget class="QWidget" name="QmitknnUNetToolGUIControls">
    <property name="geometry">
      <rect>
        <x>0</x>
        <y>0</y>
        <width>192</width>
        <height>352</height>
      </rect>
    </property>
    <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
      </sizepolicy>
    </property>
    <property name="minimumSize">
      <size>
        <width>100</width>
        <height>0</height>
      </size>
    </property>
    <property name="maximumSize">
      <size>
        <width>100000</width>
        <height>100000</height>
      </size>
    </property>
    <property name="windowTitle">
      <string>QmitknnUNetToolWidget</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
        <number>0</number>
      </property>
      <property name="topMargin">
        <number>0</number>
      </property>
      <property name="rightMargin">
        <number>0</number>
      </property>
      <property name="bottomMargin">
        <number>0</number>
      </property>
      <item>
        <layout class="QGridLayout" name="advancedSettingsLayout">
          <item row="0" column="0">
            <widget class="QLabel" name="pretraineddirLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>nnUNet Results Folder:</string>
              </property>
            </widget>
          </item>
          <item row="0" column="1" colspan="3">
            <widget class="ctkDirectoryButton" name="modeldirectoryBox" native="true" />
          </item>
          
          <!-- </item> -->
          <item row="1" column="1" colspan="3">
            <widget class="QPushButton" name="refreshdirectoryBox" native="false">
              <property name="text">
                <string>Refresh Results Folder</string>
              </property>
            </widget>
          </item>

          <!-- </item> -->
          <item row="2" column="0">
            <widget class="QLabel" name="modelLabel">
              <property name="text">
                <string>Configuration:</string>
              </property>
            </widget>
          </item>
          <item row="2" column="1">
            <widget class="ctkComboBox" name="modelBox" native="true" />
          </item>
          <item row="2" column="2">
            <widget class="QLabel" name="taskLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Task:</string>
              </property>
            </widget>
          </item>
          <item row="2" column="3">
            <widget class="ctkComboBox" name="taskBox" native="true" />
          </item>

          <!-- </item> -->
          <item row="3" column="0">
            <widget class="QLabel" name="trainerLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Trainer:</string>
              </property>
            </widget>
          </item>
          <item row="3" column="1">
            <widget class="ctkComboBox" name="trainerBox" native="true" />
          </item>

          <item row="3" column="2">
            <widget class="QLabel" name="plannerLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Plan:</string>
              </property>
            </widget>
          </item>
          <item row="3" column="3">
            <widget class="ctkComboBox" name="plannerBox" native="true" />
          </item>
          <!-- </item> -->
          <item row="4" column="0">
            <widget class="QLabel" name="foldLabel">
              <property name="text">
                <string>Fold:</string>
              </property>
            </widget>
          </item>
          <item row="4" column="1">
            <widget class="ctkCheckableComboBox" name="foldBox" native="true" />
          </item>
          <item row="4" column="3">
            <widget class="QPushButton" name="stopButton" native="false">
              <property name="text">
                <string>Stop</string>
              </property>
            </widget>
          </item>
          <!-- </item> -->
          <item row="5" column="0">
            <widget class="QLabel" name="multiModalLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Multi-Modal:</string>
              </property>
            </widget>
          </item>
          <item row="5" column="1">
            <widget class="ctkCheckBox" name="multiModalBox" native="true" />
          </item>
          <item row="5" column="2">
            <widget class="QLabel" name="multiModalSpinLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>No. of Extra Modalities:</string>
              </property>
            </widget>
          </item>
          <item row="5" column="3">
            <widget class="QSpinBox" name="multiModalSpinBox" native="true" />
          </item>

          <!-- </item> -->
          <item row="6" column="2">
            <widget class="QLabel" name="posSpinBoxLabel">
              <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                </sizepolicy>
              </property>
              <property name="text">
                <string>Reference Image Position</string>
              </property>
            </widget>
          </item>
          <item row="6" column="3">
            <widget class="QSpinBox" name="posSpinBox" native="true" />
          </item>
          <!-- </item> -->
        </layout>
      </item>

      <!-- start advanced-->
      <item>
        <widget class="ctkCollapsibleGroupBox" name="inputGroupBox" native="true">
          <property name="minimumSize">
            <size>
              <width>0</width>
              <height>0</height>
            </size>
          </property>
          <property name="collapsedHeight">
            <number>5</number>
          </property>
          <property name="title">
            <string>Advanced</string>
          </property>
          <property name="collapsed">
            <bool>true</bool>
          </property>
          <property name="flat">
            <bool>true</bool>
          </property>
          <property name="alignment">
            <set>Qt::AlignRight</set>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
            <!--  <property name="leftMargin">
              <number>0</number>
              </property>
              <property name="topMargin">
                <number>0</number>
              </property>
              <property name="rightMargin">
                <number>0</number>
              </property>
              <property name="bottomMargin">
                <number>0</number>
              </property>-->
            <item>
              <layout class="QGridLayout" name="_2">
                <property name="spacing">
                  <number>6</number>
                </property>
                <property name="margin">
                  <number>0</number>
                </property>

                <item row="0" column="0">
                  <widget class="QLabel" name="nopipLabel">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>No Pip</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="1">
                  <widget class="ctkCheckBox" name="nopipBox" native="true" />
                </item>
                <item row="0" column="2">
                  <widget class="QLabel" name="mixedPrecisionLabel">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Mixed Precision</string>
                    </property>
                  </widget>
                </item>
                <item row="0" column="3">
                  <widget class="QCheckBox" name="mixedPrecisionBox" native="true">
                    <property name="checked">
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
                <!-- </item> -->
                <item row="1" column="0">
                  <widget class="QLabel" name="gpuSpinLabel">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>GPU Id:</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="1">
                  <widget class="QSpinBox" name="gpuSpinBox" native="true" />
                </item>

                <item row="1" column="2">
                  <widget class="QLabel" name="mirrorLabel">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Enable Mirroring</string>
                    </property>
                  </widget>
                </item>
                <item row="1" column="3">
                  <widget class="ctkCheckBox" name="mirrorBox" native="true">
                    <property name="checked">
                      <bool>true</bool>
                    </property>
                  </widget>
                </item>
                <!-- </item> -->
                <item row="2" column="0">
                  <widget class="QLabel" name="pythonEnvLabel">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>Python Path:</string>
                    </property>
                  </widget>
                </item>
                <item row="2" column="1" colspan="3">
                  <widget class="ctkComboBox" name="pythonEnvComboBox" native="true" />
                </item>
                <!-- </item> -->
                <item row="3" column="0">
                  <widget class="QLabel" name="nnUnetdirLabel">
                    <property name="sizePolicy">
                      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                        <horstretch>0</horstretch>
                        <verstretch>0</verstretch>
                      </sizepolicy>
                    </property>
                    <property name="text">
                      <string>nnUNet Path:</string>
                    </property>
                  </widget>
                </item>
                <item row="3" column="1" colspan="3">
                  <widget class="ctkDirectoryButton" name="codedirectoryBox" native="true" />
                </item>
                <!-- </item> -->

              </layout>
            </item>
          </layout>
        </widget>
      </item>
      <!-- end advanced-->

      <item>
        <widget class="QPushButton" name="previewButton">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
          <property name="maximumSize">
            <size>
              <width>100000</width>
              <height>16777215</height>
            </size>
          </property>
          <property name="text">
            <string>Preview</string>
          </property>
        </widget>
      </item>
      <item>
        <widget class="QLabel" name="statusLabel">
          <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
            </sizepolicy>
          </property>
        </widget>
      </item>

    </layout>
  </widget>

  <customwidgets>
    <customwidget>
      <class>ctkDirectoryButton</class>
      <extends>QWidget</extends>
      <header location="global">ctkDirectoryButton.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">ctkComboBox.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkCheckableComboBox</class>
      <extends>QComboBox</extends>
      <header location="global">ctkCheckableComboBox.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkCheckBox</class>
      <extends>QCheckBox</extends>
      <header location="global">ctkCheckBox.h</header>
      <container>1</container>
    </customwidget>
    <customwidget>
      <class>ctkCollapsibleGroupBox</class>
      <extends>QGroupBox</extends>
      <header location="global">ctkCollapsibleGroupBox.h</header>
      <container>1</container>
    </customwidget>
  </customwidgets>
  <resources />
  <connections />
</ui>
